README.md

```diff
@@ -6,62 +6,69 @@
 Welcome
 =======
 
-This is the Cyrus IMAP Server, version series 3.0.x.
+This is the Cyrus IMAP Server, developer version 3.5.  This version is under
+active development, and is not considered "stable".
 
-No further development work will progress on anything older than version 2.3.
-Versions 2.3 and 2.4 still receive security updates, but new features or
-non-security bugfixes are unlikely to be backported.  Version 2.5 still
-receives security updates and non-security bugfixes. Version 3.0 is under
-active development.
+The current stable series is 3.4.
+
+Versions 3.0 and 3.2 still receive security updates, and some non-security
+bug fixes.
 
 What is Cyrus
 =============
 
-Cyrus is an IMAP server, where IMAP (Internet Message Access Protocol) 
+Cyrus is an IMAP server, where IMAP (Internet Message Access Protocol)
 is a protocol for accessing mail.
 
-The Cyrus IMAP server differs from other IMAP server implementations in 
-that it is generally intended to be run on "sealed" servers, where 
-normal users are not permitted to log in. The mailbox database is stored 
-in parts of the filesystem that are private to the Cyrus IMAP system. 
-All user access to mail is through the IMAP, NNTP, or POP3 protocols. 
+The Cyrus IMAP server differs from other IMAP server implementations in
+that it is generally intended to be run on "sealed" servers, where
+normal users are not permitted to log in. The mailbox database is stored
+in parts of the filesystem that are private to the Cyrus IMAP system.
+All user access to mail is through the IMAP, NNTP, or POP3 protocols.
```

Comment:
and JMAP!  It's an RFC protocol.

Cyrus also has an HTTP server and CalDAV/CardDAV implementations.